Makefile

```diff
@@ -0,0 +1,54 @@
+all: build test check-coverage lint  ## test, check coverage and lint
+	@if [ -e .git/rebase-merge ]; then git --no-pager log -1 --pretty='%h %s'; fi
+	@echo '$(COLOUR_GREEN)Success$(COLOUR_NORMAL)'
+
+clean::  ## Remove generated files
+
+.PHONY: all clean
+
+# --- Build and lint ---
+
+build: ## Build servedir binary
+	go build -o servedir .
```

Comment:
you can leave out the `-o servedir` - that is the default name from the directory. I think you can also leave off the `.`, yes?

Reply:
stupid github doubled this comment when I edited it to add the extra bit.

Reply:
done.